CONTRIBUTING.md

```diff
@@ -208,4 +208,14 @@ Alternatively, to start a jupyter notebook, there are two steps, first run:
 
 This will output something similar to `http://(<docker container id> or <ip>):8888/?token=<token id>`, and can be accessed at `http://localhost:8888/?token=<token id>`.
 
+## Running the benchmark tests
+
+To run the **benchmark tests** do the following:
+
+- Install airspeed velocity via `pip install asv`
+
+- `cd arviz/`
+
+- `asv run`
```

Comment:
what about?

    $ cd arviz/
    $ asv run

minor comment, it is only to follow the convention from the rest of the contributing file.

Reply:
Sure, and I'll change ELPD data too